### PR TITLE
[DO-388] Add removal UI to multi select selected values

### PR DESCRIPTION
**Pull request by sbone**
WIP
options

### CODE REVIEW

**Comment by sbone** [comment]
@jachin I have this functionality working! Going to connect w/ Jon on styling next week.

File: src/Main.elm [hunk 1]
```diff
@@ -55,47 +55,9 @@ import Option
...
-        , selectOptionReceiver
-        , selectedItemStaysInPlaceChangedReceiver
-        , valueCasingDimensionsChangedReceiver
-        , valueChanged
-        , valueChangedReceiver
-        , valueCleared
-        , valueDecoder
-        , valuesDecoder
-        )
+import Ports exposing (addOptionsReceiver, allOptions, allowCustomOptionsReceiver, blurInput, customOptionHintReceiver, customOptionSelected, deselectOptionReceiver, disableChangedReceiver, errorMessage, focusInput, inputBlurred, inputKeyUp, loadingChangedReceiver, maxDropdownItemsChangedReceiver, muchSelectIsReady, multiSelectChangedReceiver, multiSelectSingleItemRemovalChangedReceiver, optionDeselected, optionSelected, optionsChangedReceiver, placeholderChangedReceiver, removeOptionsReceiver, requestAllOptionsReceiver, scrollDropdownToElement, selectOptionReceiver, selectedItemStaysInPlaceChangedReceiver, valueCasingDimensionsChangedReceiver, valueChanged, valueChangedReceiver, valueCleared, valueDecoder, valuesDecoder)
```
Comment (jachin) [inline src/Main.elm:58]: You could break up these across multiple lines.

File: src/much-select.js [hunk 2]
```diff
@@ -305,6 +311,10 @@ class MuchSelect extends HTMLElement {
       if (oldValue !== newValue) {
         this.isInMultiSelectMode = newValue;
       }
+    } else if (name === "multi-select-single-removable-options") {
```
Comment (jachin) [inline src/much-select.js:314]: In your example you call the attribute `single-item-removal`... are you going with `multi-select-single-removable-options`?

Comment (jachin) [reply src/much-select.js:314]: You probably want to add the attribute to the `observedAttributes` too.

**Review by jachin — CHANGES_REQUESTED** [review]
I have a couple things that should probably be changed/fixed. Thank you for all the spelling fixes.

**Review by jachin — APPROVED** [review]
Shoot, I'm sorry, I thought I approved this yesterday.

**Pull request merged**